peterpy/database/connection.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+from sqlalchemy import Engine, create_engine
+from sqlalchemy.orm import Session
+from peterpy.config import config
+
+connection_string = f"mysql+mysqlconnector://root:{config["MYSQL_ROOT_PASSWORD"]}@{config["MYSQL_HOST"]}:{config["MYSQL_TCP_PORT"]}/{config["MYSQL_DATABASE"]}"
+
+
+class DatabaseConnection:
+    def __init__(self):
+        try:
```

Comment:
Wouldn't do a try catch here. Now you might get an app that doesn't do anything without it being clear. You want to have the app crash completely if it can't create the connection

Reply:
removed it here and try/ catch now in main(), don't start the app when db connection fails